includes/amp-helper-functions.php

```diff
@@ -583,26 +583,26 @@ function amp_get_content_embed_handlers( $post = null ) {
 	 */
 	return apply_filters( 'amp_content_embed_handlers',
 		array(
-			'AMP_Core_Block_Handler'        => array(),
-			'AMP_Twitter_Embed_Handler'     => array(),
-			'AMP_YouTube_Embed_Handler'     => array(),
-			'AMP_DailyMotion_Embed_Handler' => array(),
-			'AMP_Vimeo_Embed_Handler'       => array(),
-			'AMP_SoundCloud_Embed_Handler'  => array(),
-			'AMP_Instagram_Embed_Handler'   => array(),
-			'AMP_Issuu_Embed_Handler'       => array(),
-			'AMP_Meetup_Embed_Handler'      => array(),
-			'AMP_Vine_Embed_Handler'        => array(),
-			'AMP_Facebook_Embed_Handler'    => array(),
-			'AMP_Pinterest_Embed_Handler'   => array(),
-			'AMP_Playlist_Embed_Handler'    => array(),
-			'AMP_Reddit_Embed_Handler'      => array(),
-			'AMP_Tumblr_Embed_Handler'      => array(),
-			'AMP_Gallery_Embed_Handler'     => array(),
-			'AMP_Gfycat_Embed_Handler'      => array(),
-			'AMP_Hulu_Embed_Handler'        => array(),
-			'AMP_Imgur_Embed_Handler'       => array(),
-			'WPCOM_AMP_Polldaddy_Embed'     => array(),
+			'AMP_Core_Block_Handler'                  => array(),
+			'AMP_Twitter_Embed_Handler'               => array(),
+			'AMP_YouTube_Embed_Handler'               => array(),
+			'AMP_DailyMotion_Embed_Handler'           => array(),
+			'AMP_Vimeo_Embed_Handler'                 => array(),
+			'AMP_SoundCloud_Embed_Handler'            => array(),
+			'AMP_Instagram_Embed_Handler'             => array(),
+			'AMP_Issuu_Embed_Handler'                 => array(),
+			'AMP_Meetup_Embed_Handler'                => array(),
+			'AMP_Vine_Embed_Handler'                  => array(),
+			'AMP_Facebook_Embed_Handler'              => array(),
+			'AMP_Pinterest_Embed_Handler'             => array(),
+			'AMP_Playlist_Embed_Handler'              => array(),
+			'AMP_Reddit_Embed_Handler'                => array(),
+			'AMP_Tumblr_Embed_Handler'                => array(),
+			'AMP_Gallery_Embed_Handler'               => array(),
+			'AMP_Gfycat_Embed_Handler'                => array(),
+			'AMP_Hulu_Embed_Handler'                  => array(),
+			'AMP_Imgur_Embed_Handler'                 => array(),
+			'WPCOM_AMP_Polldaddy_Embed'               => array()
 		),
```

Comment:
Can you please remove the tab/comma changes here to revert back to the original file.?

Reply:
ohh yes